.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## Pull Request Template for pycountry
+
+Thank you for contributing to pycountry! Before submitting this PR, please ensure you've completed the following steps:
+
+### Pre-Submission Checklist:
+- [ ] Code is properly formatted and adheres to the project's coding guidelines.
+- [ ] Changes have been tested with the latest version of the codebase.
+- [ ] Code changes do not decrease the current test coverage.
+- [ ] All existing tests pass successfully.
+- [ ] New tests (if applicable) are added to cover your changes.
+
+### Local Testing:
+Please run the following commands to ensure your changes do not introduce any new issues:
+- [ ] `pytest` - Runs all the tests to ensure they pass.
+- [ ] `make test` - Executes the custom test suite.
+- [ ] `mypy` - Checks for type consistency.
+- [ ] `pre-commit run --all-files` - Ensures code adheres to pre-commit checks.
```

Comment:
We should bundle all of these into a `make check` target.

Reply:
@zware does that just require us combining the mypy-check, pre-commit, and test.yml file into a check.yml file?

Reply:
Done!

Reply:
No need to change the CI checks for this, just the `Makefile` target you added :)